math/genvector/inc/Math/GenVector/Cartesian2D.h

```diff
@@ -89,8 +89,9 @@ public :
    Scalar X()     const { return fX;}
    Scalar Y()     const { return fY;}
    Scalar Mag2()  const { return fX*fX + fY*fY; }
-   Scalar R()     const { return std::sqrt( Mag2());}
-   Scalar Phi()   const { return (fX==0 && fY==0) ? 0.0 : atan2(fY,fX);}
+   Scalar R()     const { using namespace std; return sqrt( Mag2()); }
```

Comment:
Btw, it's @lmoneta's call but I'd prefer to have a short comment saying what the pattern does, or maybe a #define R__enable_vc_types using namespace std; This would make this easier to read by an unarmed eye.